### PR TITLE
Move affiliate links disclaimer into article meta

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Introduce a new component representing the affiliate links disclaimer. This appears on pages that contain affiliate links provided by our partner Skimlinks.

The disclaimer appears below the standfirst. This is a change to the previous location, which was at the end of the article, as the last element in the list of page elements.

## Why?

For legal reasons, the affiliate links disclaimer needs to appear above the content of the article. There are several ways of achieving this, which include:

- Under the standfirst
- Below the main media
- Prepending to the list of page (block) elements

We chose to move it under the standfirst, as it looks consistently okay across breakpoints and article types

## Related PRs

- [x] guardian/frontend#26727
- [ ] guardian/frontend#26738
- [x] Historic change that added the Disclaimer to page elements: guardian/frontend#21119

## TODO - following launch
- [ ]  Remove disclaimer from page elements in frontend's DotcomRenderingModel as no longer needed
- [ ] Delete DisclaimerBlockComponent, associated logic and types from DCR as no longer needed

### Standard article

| Mobile      | Desktop      |
|-------------|------------|
| ![standard-mobile][] | ![standard-desktop][] |

[standard-mobile]: https://github.com/guardian/dotcom-rendering/assets/5931528/7226fcbd-9da4-49b7-a871-14bf511211da
[standard-desktop]: https://github.com/guardian/dotcom-rendering/assets/5931528/97fbc975-7f6b-40a6-8b9f-bb84b5487b5f

### Showcase

| Mobile      | Desktop      |
|-------------|------------|
| ![showcase-mobile][] | ![showcase-desktop][] |

[showcase-mobile]: https://github.com/guardian/dotcom-rendering/assets/5931528/b79a524f-a9d1-4e0d-86f5-2f6a8fb059be
[showcase-desktop]: https://github.com/guardian/dotcom-rendering/assets/5931528/ec63d7bb-0880-4ffa-b580-c7de7e69d269

### Immersive

| Mobile      | Desktop      |
|-------------|------------|
| ![immersive-mobile][] | ![immersive-desktop][] |

[immersive-mobile]: https://github.com/guardian/dotcom-rendering/assets/5931528/5253ac74-849f-4cb5-b1f9-a5ae2960ea1d
[immersive-desktop]: https://github.com/guardian/dotcom-rendering/assets/5931528/9866a25e-9bee-4bb4-9d77-5a79242571f8

### CODE REVIEW

**Comment by SiAdcock** [comment]
The failing Cyrpess test is due to the DotcomRenderingModel changing. The `affilateLinksDisclaimer` will be available when guardian/frontend#26727 is merged, and I expect the Cypress test will pass as a result.

```
  1) Elements
       WEB
         should render the affiliate disclaimer block:
     AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-testid="affiliate-disclaimer"]`, but never found it.
```

File: dotcom-rendering/src/layouts/ShowcaseLayout.tsx [hunk 1]
```diff
@@ -551,6 +559,15 @@ export const ShowcaseLayout = (props: WebProps | AppsProps) => {
 								standfirst={article.standfirst}
 							/>
 						</GridItem>
+						<GridItem area="disclaimer">
+							{hasAffiliateLinksDisclaimer && (
+								<Disclaimer
+									html={
+										article.affiliateLinksDisclaimer ?? ''
```
Comment (chrislomaxjones) [inline dotcom-rendering/src/layouts/ShowcaseLayout.tsx:566]: Feel free to ignore this one, but I think if you inlined the definition of `hasAffiliateLinksDisclaimer` you would't have to include the `?? ''`

**Review by chrislomaxjones — APPROVED** [review]
Nice one!

File: dotcom-rendering/src/components/Elements.amp.tsx [hunk 2]
```diff
@@ -360,6 +351,8 @@ export const Elements = (
 						adTargeting={adTargeting}
 					/>
 				);
+			case 'model.dotcomrendering.pageElements.DisclaimerBlockElement':
+				return <></>;
```
[the remark this answers is not in the document]
Comment (SiAdcock) [reply dotcom-rendering/src/components/Elements.amp.tsx:355]: Yes that's right, just belt and braces really. I intend to get rid of it post-launch.

**Comment by HarryFischer** [comment]
@SiAdcock could I ask for a couple tweaks...

- max width to match the width of standfirst (80% off top of my head but worth checking)
- line height 130%
- font style normal and font family text sans

Let me know!

**Comment by SiAdcock** [comment]
> @SiAdcock could I ask for a couple tweaks...
> 
> * max width to match the width of standfirst (80% off top of my head but worth checking)
> * line height 130%
> * font style normal and font family text sans
> 
> Let me know!


Thanks for the suggestions @HarryFischer 🙏 ❤️  How does this look?

Screenshot 2023-12-05 at 16 00 42

When you say "font style normal", did you mean not italic? If so, I'll need to make an accompanying change in frontend, as this block is wrapped in an `<em>` which is making it italic. All other changes are included in this PR.

**Pull request merged**